<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <comment>
    * Released as open source by Gabriel Caudrelier
    *
    * Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com
    *
    * https://github.com/metrodango/pip3line
    *
    * Released under BSD 3-Clause see LICENSE for more information
  </comment>
 <class>DeleteableListItem</class>
 <widget class="QWidget" name="DeleteableListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>60</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="pixLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>text</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deletePushButton">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Images/icons/dialog-cancel-3.png</normaloff>:/Images/icons/dialog-cancel-3.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
